{{cookiecutter.project_slug}}/.nvmrc

```diff
@@ -0,0 +1 @@
+lts/carbon
```

Comment:
maybe little descriptions of what is for?

Reply:
https://www.notion.so/developersociety/Javascript-edc8652acc914dd8864c8294c434fb77

Currently a work in progress - want to ensure that all our tools work with nvm before getting everyone to upgrade to it. Until then sticking to lts/carbon (node v8) means that anyone without nvm just yet will be fine.